ang/civicase/ActivityActions--tags.html

```diff
@@ -0,0 +1,24 @@
+<div id="bootstrap-theme" class="civicase__tags-modal">
+  <form class="form-horizontal">
+    <div class="form-group civicase__tags-modal__generic-tags__container">
```

Comment:
Isn't the BEM depth too deep? It's already 4 levels. Can we rename the class to `civicase__tags-modal__tags-container` or `civicase__tags-modal__tags-container`

Reply:
Renamed to `civicase__tags-modal__generic-tags-container`, which decreases one level.
This container is only for the generic tags, so I kept the `generic` word. is it ok?